internal/registry/image.go

```diff
@@ -50,6 +50,11 @@ func ExtractBundleImage(ctx context.Context, logger *log.Entry, image string, lo
 	// Export the image into bundleDir.
 	logger = logger.WithFields(log.Fields{"dir": bundleDir})
 
+	//if user set --skip-tls then set --skip-tls-verify to true as --skip-tls is deprecated
+	if skipTLS {
+		skipTLSVerify = true
+	}
+
```

Comment:
This check should be done much earlier in the process, closer to where cobra is parsing the flag.

Reply:
Removed it.